docs/UserGuide.md

```diff
@@ -62,116 +62,68 @@ Jobby comes equipped with a [GUI]() interface that provides visual feedback to y
-
-![ui overview](images/ug-images/labelled-gui.png)
-
-### Adding `Contact`
-Jobby can create `Contacts`. In the form of `Organization` and `Recruiter`, which you can add into your database with simple CLI commands.
-
-Let's say you want to track your job application for the `Organization: Woogle`, with a `Recruiter: John`.
-
-You can add the `Organization` into Jobby with a simple command:
-```add --org --name Woogle --id woogle_id```
-
-![Add Organization](images/starter-guide/add-woogle.jpg)
-
-
-You can then add a **recruiter** that's related to the "Woogle" organization like so::
-```add --rec --name John --id john_id --oid woogle_id```
-
-![Add Recruiter](images/starter-guide/add-john.jpg)
-
-
-> To find out all of `Organization` and `Recruiter`'s flags, go [organization documentation]()
-> and [recruiter documentation]()
-
-### Adding `Application`
-Let's say now you want to apply to `Woogle` for the position of `Junior Software Engineer` role. You will be able to create
-an `Application` for this company with the by including the `Organization ID` in the command:
-`apply woogle_id --title Software Engineer --by 22-11-2023`
-
-![Add Application](images/starter-guide/add-application.jpg)
+### The Beginning
+When you first launch Jobby, you will notice that it comes preloaded with sample data for you to play with. You may use 
+the sample data to familiarise yourself with the various [features](#features) that Jobby provides.
 
-`Application` will automatically generates the fields if there are no inputs on your end.
-- `Status: Pending`
-- `Stage: Resume`
+Once you're ready to start using Jobby for your internship tracking, you can easily remove the sample data:
+1. type `clear` into the command box
+2. Hit `enter`
 
-> You will be able to add different details of `Application`, see the [applcation documentation]() here.
+Solid! Now it's time to get started with Jobby!
 
-### Editing `Contact`
-Now that you have both your Contacts and Applications. You received new informations about your `Contacts`, you will have to edit your `Contact` data.
+### Adding your first Organization
 
-We can do so with the `edit` CLI command with the desired id and input.
+Let's say you are interested to applying to **Google** as your internship destination, and you found their email **google@gmail.com**.
 
-Let's say we want to change the phone detail of `Organization: Woogle` and email of `Recruiter: John`.
-We can use the commands:
-`edit woogle_id --phone 12345678` and `edit john_id --email john@gmail.com`
+You can use [add --org](#adding-organizations---add---org) command here to add Google into your organization:
+1. type `add --org --name Google --id google_id --email google@gmail.com` into the command box
+2. press `enter`
 
-![Edit Contact](images/starter-guide/edit-john.jpg)
+![Adding Organization](images/ug-images/org-added.png)
 
+You have successfully added **Google**, with the email **google@gmail.com** into your **organization contact**!
```

Comment:
Since you're suggesting the organization Google, it is unlikely for them to have the consumer domain `@gmail.com`. Rather, their emails would more likely be variants of `@*.google.com`. It might thus be better to use a more plausible email for this to make the guide sound more convincing.

Reply:
I think it's fine for it to be in Gmail as it's very much associated with them

Reply:
I alr added a mini disclaimer that of course it is not their email